scripts/openstack-quickstart-demosetup

```diff
@@ -1055,6 +1055,16 @@ EOF
         ansible-playbook -i monasca-hosts monasca.yml
     )
 
+    # do the agent setup
+    monasca-setup \
+        -u "admin" \
+        -p "$pw" \
+        --skip_enable \
+        --user "monasca-agent" \
```

Comment:
Sumit and me just dug through our local working copies of openstack-quickstart and there are two lines missing here (they must have gotten dropped during all that multiple-pull-requests-with-various-rebases craziness):

```
--log_dir /var/log/monasca-agent/ \
--project_name admin \
```

Without these monasca-agent is unlikely to run/work.